index.js

```diff
@@ -47,14 +41,19 @@ async function init() {
     // it's fine if the directory exists - but throw on anything else
     if(ex.code !== 'EEXIST' /* directory already exists */) { throw ex; }
   }
-  
+
+  // pull in info about the site
   const infoFile = await readFile(path.join(__dirname, 'src/info.json'));
   const infoJSON = {
     ...JSON.parse(infoFile.toString()),
     generatedDate: new Date().toISOString(),
     resources: [],
```

Comment:
this should probably use any default resources from the json file, so
```js
  const infoJSON = {
    resources: [],
    ...JSON.parse(infoFile.toString()),
    generatedDate: new Date().toISOString(),
  };
```

Reply:
This is a good catch!